wolfcrypt/src/rsa.c

```diff
@@ -1031,11 +1013,22 @@ static int wc_RsaFunctionAsync(const byte* in, word32 inLen, byte* out,
     case RSA_PRIVATE_DECRYPT:
     case RSA_PRIVATE_ENCRYPT:
     #ifdef HAVE_CAVIUM
-        ret = NitroxRsaExptMod(in, inLen, key->d.dpraw, key->d.used,
-                               key->n.dpraw, key->n.used, out, outLen, key);
+        ret = NitroxRsaExptMod(in, inLen,
+                               key->d.raw.buf, key->d.raw.len,
+                               key->n.raw.buf, key->n.raw.len,
+                               out, outLen, key);
     #elif defined(HAVE_INTEL_QA)
-        /* TODO: Add support for Intel Quick Assist */
-        ret = -1;
+        #ifdef RSA_LOW_MEM
```

Comment:
I agree it looks like there should be a way to reduce the RsaKey structure if RSA_LOW_MEM is defined. The RSA_LOW_MEM only uses d, n and e. The CRT mode uses p, dP, q, dQ and u. The key gen builds all these fields so I am not sure how to have it just do d, n and e. I'll make note for a separate PR altogether since this applies to current master as well, not just async.